pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -912,3 +938,7 @@ func (ClosedDoorSecurityPolicyEnforcer) EnforceMountPolicy(_, _ string, _ *oci.S
 func (ClosedDoorSecurityPolicyEnforcer) ExtendDefaultMounts(_ []oci.Mount) error {
 	return nil
 }
+
+func (ClosedDoorSecurityPolicyEnforcer) EnforceLogging() error {
+	return nil
+}
```

Comment:
the closed door shouldnt allow logging. ClosedDoor is "deny everthing"

Reply:
are you suggesting that I also set `io.Discard` here as well?

Reply:
No. I don't know the correct mechanism, but closed door is "deny everything" so if open door allows logging, then closed door should deny it as closed door is the mirror opposite.

Reply:
that makes sense.